src/loaders/lottie/tvgLottieModel.h

```diff
@@ -456,7 +457,10 @@ struct LottieSolidFill : LottieObject
 };
 
 
-struct LottieGradientFill : LottieObject, LottieGradient
+struct LottieGradientObject : LottieObject, LottieGradient {};
```

Comment:
Can't see any necessity of LottieGradientObject in this commit context. Could you please explain this?

Reply:
In this change, `parseProperty` will have parameter `LottieObject` to parse `sid` into itself.

When system calls `parseProperty` from `parseGradient`:
```cpp
void LottieParser::parseGradient(LottieGradient* gradient, const char* key)
{
    // ...
    // Can't pass original lottie object because LottieGradient isn't child of LottieObject for this scope.
    else if (!strcmp(key, "o")) parseProperty(gradient->opacity, gradient); // [!] Type error
    // ...
}
```

Even though, `parseGradient` is called with LottieObject such as
- LottieGradientFill
- LottieGradientStroke

the parameter is being type-changed because of the explicit type of `LottieGradient`, it wasn't able to pass as `LottieObject`.


But if we declare `LottieGradientObject`, the `LottieObject` will be correctly passed.
```cpp
// the `gradient` is matched the `LottieObject`
void LottieParser::parseGradient(LottieGradientObject* gradient, const char* key)

// template<typename T> void parseProperty(T& prop, LottieObject* parent)
```